docs/RuntimeAttributes.md

```diff
@@ -104,61 +133,63 @@ And the effective runtime for `task second` is:
 
 Note how for `task second` the WDL value for `docker` is used instead of the default provided in the workflow options.
 
-## `continueOnReturnCode`
 
-When each task finishes it returns a code. Normally, a non-zero return code indicates a failure. However you can override this behavior by specifying the `continueOnReturnCode` attribute.
+## Runtime Attribute Descriptions
 
-When set to false, any non-zero return code will be considered a failure. When set to true, all return codes will be considered successful.
+### `cpu`
 
-```
-runtime {
-  continueOnReturnCode: true
-}
-```
+*Default: _1_*
 
-When set to an integer, or an array of integers, only those integers will be considered as successful return codes.
+`cpu` represents the number of cores that a job should be allocated.
```

Comment:
"that a job should be allocated" ... something needs to change in one direction or another

Reply:
Rewrote it (8cc7be9984f9ffad102ba2ad62e91925ef0caeb0) to align closer with what memory says:

> The `cpu` runtime attribute represents the number of cores that a job requires, however each backend may interpret this differently: